Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
+
+        /*
+         * Routes:
+         * Here you can register new routes in your Bolt application.
+         * The first route will be handles in this Extension class,
+         * then we switch to an extra Controller class for the routes.
+         * */
+
+        // register route for all GET requests on '/example/url' that will be handled in this class ($this) in the 'routeExampleUrl' function
+        $this->app->get("/example/url", array($this, 'routeExampleUrl'))->bind('example-url');
```

Comment:
Long fluent interface use is easier to read if broken up, e.g.
```php
        $this->app
            ->get('/example/url', array($this, 'routeExampleUrl'))
            ->bind('example-url');
```

Also please only use `'`single quotes`'` unless needed :grin: